.github/CODEOWNERS

```diff
@@ -6,4 +6,4 @@
 
 
 # The yoshi-nodejs team is the default owner for nodejs repositories.
-*     @googleapis/yoshi-nodejs
+*     @googleapis/yoshi-nodejs @googleapis/api-dataproc
```

Comment:
Forgive me - this is controlled now in `.repo-metadata.json` in the root of the repo, and it uses that to generate the `CODEOWNERS` file now.  Kinda like this:
https://github.com/googleapis/nodejs-datastore/blob/master/.repo-metadata.json#L12

Reply:
Does it? I don't see anything similar here: https://github.com/googleapis/nodejs-dataproc/blob/master/.repo-metadata.json even for the existing CODEOWNER. How should we structure this if there are multiple teams here, e.g. Yoshi as primary and Dataproc team as secondary?

Reply:
The @googleapis/yoshi-nodejs team should always be added, with an optional second team: